plugins/httpboot/plugin.go

```diff
@@ -54,9 +69,26 @@ func setup4(args ...string) (handler.Handler4, error) {
 
 func Handler6(req, resp dhcpv6.DHCPv6) (dhcpv6.DHCPv6, bool) {
 	log.Debugf("Received DHCPv6 request: %s", req.Summary())
+
+	var ukiURL string
+	if !useBootService {
+		ukiURL = bootFile6
+	} else {
+		clientIPs, err := extractClientIP6(req)
+		if err != nil {
+			log.Errorf("failed to extract ClientIP, Error: %v Request: %v ", err, req)
+			return resp, true
```

Comment:
Change to false, so if your plugin fails for whatever reason, the plugin chain will not break (meaning: the next plugin in the configuration will get executed)